openassessment/xblock/grade_mixin.py

```diff
@@ -361,7 +361,7 @@ def _get_assessment_part(title, feedback_title, part_criterion_name, assessment)
         )
         self_assessment_part = _get_assessment_part(
             _('Self Assessment Grade') if is_staff else _('Your Self Assessment'),
-            _('Self Comments'),
+            _('Your Comments'),
```

Comment:
This is in order to match the heading on overall feedback, I had it wrong initially

<img width="673" alt="screen shot 2016-04-14 at 10 26 37 am" src="https://cloud.githubusercontent.com/assets/7373924/14531480/77bd85f8-022b-11e6-843e-0e3757361f20.png">